cmd/test/main.go

```diff
@@ -79,6 +80,13 @@ func New(ctx *TestCommandContext) *cobra.Command {
 		Use:   "test <pattern>",
 		Short: "Execute static analysis for given <pattern>",
 		Long:  "Execute static analysis for given <pattern>. Input should be glob or `-` for stdin",
+		Example: utils.Example(`
```

Comment:
There should also be reference here to sending manifests to datree through stdin.
for example: `cat kube-prod/deployment.yaml | datree test -`